virtusize/src/main/java/com/virtusize/android/ui/VirtusizeWebViewFragment.kt

```diff
@@ -121,7 +122,7 @@ class VirtusizeWebViewFragment : DialogFragment() {
                                     return true
                                 }
                             }
-                            return false
+                            return VirtusizeAuth.isSNSAuthUrl(this@VirtusizeWebViewFragment, url)
```

Comment:
Please refer to https://github.com/virtusize/virtusize-auth-android/tree/feature/facebook-auth